packages/server/src/synchronizer/pipeline/rules/rpc/index.ts

```diff
@@ -1,15 +1,29 @@
 import File from 'vinyl'
+import {relative, resolve} from 'path'
+import {pathExistsSync} from 'fs-extra'
+
 import {absolutePathTransform} from '../../utils'
-import {relative} from 'path'
 import {through} from '../../../streams'
 import {Rule} from '../../../types'
 
+const configFiles = ['next.config.js', 'blitz.config.js']
+
 /**
  * Returns a Rule that manages generating the internal RPC commands and handlers
  */
 const create: Rule = function configure({config: {src}}) {
   const fileTransformer = absolutePathTransform(src)
 
+  let serverless = false
+  for (const configFile of configFiles) {
+    if (pathExistsSync(resolve(src, configFile))) {
+      const config = require(resolve(src, configFile))
+      if (Object.keys(config).includes('target') && config.target.includes('serverless')) {
+        serverless = true
+      }
+    }
+  }
+
```

Comment:
I think I would be more comfortable if we did elsewhere possibly as a global singleton config loader. I am pretty sure there is a package out there to handle this.

Reply:
Happy to merge with a big TODO and then open an issue for it?

Reply:
Ok I was thinking of https://github.com/davidtheclark/cosmiconfig - this has ramifications to the config rule so best addressed in a separate PR

Reply:
Here is an issue https://github.com/blitz-js/blitz/issues/332